### PR TITLE
Add robust04 data generation code for reranking

### PR DESCRIPTION
**Pull request by Victor0118**
@lintool Could you take a look at this PR?

### CODE REVIEW

File: src/main/python/rerank/scripts/export_robust04_dataset.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            method = "BM25_0.9_0.5"
+        else:
+            print("Unsupported ranking method")
+            break 
+        prediction_fn = "predict_{}_robust04_split{}_{}.txt".format(method, split, mode)
+        output_dir = os.path.join(args.output_dir, "Robust04Corpus")
+        output_fn = os.path.join(output_dir, "split{}_{}_{}.txt".format(split, mode, method))
+        if not os.path.exists(output_dir):
+            os.makedirs(output_dir)
+            searcher.setBM25Similarity(0.9, 0.5)
```
Comment (lintool) [inline src/main/python/rerank/scripts/export_robust04_dataset.py]: I think BM25 defaults are 0.9, 0.4: https://github.com/castorini/Anserini/blob/master/src/main/java/io/anserini/search/SearchArgs.java#L116

We should match defaults?

File: src/main/python/rerank/scripts/export_robust04_dataset.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+parser.add_argument('--output_dir', default='src/main/python/rerank/data', help='output directory')
+parser.add_argument('--K', default=1000, type=int, help='number of document retrieved')
+parser.add_argument('--index', default="/tuna1/indexes/lucene-index.robust04.pos+docvectors+rawdocs", help='index of Robust04 corpus')
+args = parser.parse_args()
+
+searcher = JSearcher(JString(args.index))
+
+fqrel = "src/main/resources/topics-and-qrels/qrels.robust2004.txt"
+qid2reldocids = get_qid2reldocids(fqrel)
+best_rm3_parameters = [[47, 9, 0.3], [47, 9, 0.3], [47, 9, 0.3], [47, 9, 0.3], [26, 8, 0.3]]
```
Comment (lintool) [inline src/main/python/rerank/scripts/export_robust04_dataset.py]: Add a comment here about these being the parameters for five-fold from SIGIR Forum 2018 paper.

Comment (Victor0118) [reply src/main/python/rerank/scripts/export_robust04_dataset.py]: Okay.

File: src/main/python/rerank/scripts/export_robust04_dataset.py (hunk 1 above)
Comment (Victor0118) [reply src/main/python/rerank/scripts/export_robust04_dataset.py]: Right. Thanks!

**Pull request merged**